cranelift-interpreter/src/value.rs

```diff
@@ -0,0 +1,45 @@
+//! The values operated on by the interpreter.
+//!
+//! Each variant of `Value` may implement traits of `std::ops` for use in `interpreter`; e.g.,
+//! `Value::Int` has an implementation (a match arm) of `Add`.
+
+use std::ops::{Add, Sub};
+
+#[derive(Clone, Debug, PartialEq)]
+pub enum Value {
+    Bool(bool),
+    Int(i64),
```

Comment:
I think this should be `u64` instead. In any case it is important to not forget correct zero/sign extension.

Reply:
You're probably right; let me think some more about this.